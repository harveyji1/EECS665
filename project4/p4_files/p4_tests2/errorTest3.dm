b : int = a + 1;
