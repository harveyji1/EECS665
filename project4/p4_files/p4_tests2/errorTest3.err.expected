FATAL [1,11]-[1,12]: Undeclared identifier
Name Analysis Failed
